demisto_sdk/validation/file_validator.py

```diff
@@ -310,9 +311,15 @@ def validate_added_files(self, added_files):
                 if self.id_set_validator.is_file_has_used_id(file_path):
                     self._is_valid = False
 
-            if re.match(TEST_PLAYBOOK_REGEX, file_path, re.IGNORECASE):
-                if not self.conf_json_validator.is_test_in_conf_json(collect_ids(file_path)):
+            returned_playbook_regex = checked_type(file_path, PLAYBOOKS_REGEXES_LIST, re.IGNORECASE)
```

Comment:
don't think we need the re.IGNORECASE here, should be True

Reply:
removed this line, see the updated file